docs/learn/guides/database-integrations/mongodb_integrations/insert_data/README.md

```diff
@@ -0,0 +1,102 @@
+# Reading data from a CSV file and inserting to MongoDB
+
+## About 
+Ballerina is an open-source programming language that supports developers to integrate their system easily with the support of connectors. In this guide, we are mainly focusing on importing CSV file having contacts into MongoDB using FTP connector.
+
+`wso2/mongodb` module allows you to perform CRUD operations on Mongo DB.<br/> 
+The `wso2/ftp` module provides a FTP client and a FTP server listener implementation to facilitate an FTP connection 
+to a remote location. You can find other integrations modules from the [wso2-ballerina](https://github.com/wso2-ballerina) Github repository. 
+
+## What you'll build
+
+This application listens to a remote FTP location and when a CSV file is added to that FTP location, it will fetch the CSV file, reads its content and insert the content into Mongo DB. Then a 
+success message is logged if the operation is successful, .
+
+![inserting csv data to mongo db](../../../../../assets/img/mongo_insert.png)
+
+## Prerequisites
+Link to download ballerina integrator.
```

Comment:
Don't you need VS Code, MongoDB, and Java we well?

Reply:
IMO, as those are common to every guide we should not include them here.